File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        
        String[] numbers = s.split(" ");
        int min = Integer.MAX_VALUE; 
        int max = Integer.MIN_VALUE; 

        for (String num : numbers) {
            int value = Integer.parseInt(num);
            if (value < min) {
                min = value;
            }
            if (value > max) {
                max = value;
            }
        }

        return min + " " + max;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    # 최소값과 최대값을 찾아 문자열 형태로 return
    answer = ''
    
    s = list(map(int,s.split()))
    print(s)
    MAX = max(s)
    MIN = min(s)
    
    answer = MAX + MIN
    
    return f"{MIN} {MAX}"